packages/generator/src/generators/app-generator.ts

```diff
@@ -181,16 +181,26 @@ export class AppGenerator extends Generator<AppGeneratorOptions> {
   }
 
   commitChanges() {
+    const commitSpinner = log.spinner(log.withBrand("Committing your app")).start()
     const commands: Array<[string, string[], object]> = [
       ["git", ["add", "."], {stdio: "ignore"}],
-      ["git", ["commit", "-m", "New baby Blitz app!"], {stdio: "ignore"}],
+      [
+        "git",
+        ["commit", "--no-gpg-sign", "-m", "New baby Blitz app!"],
+        {stdio: "ignore", timeout: 5000},
```

Comment:
any particular reason for the 5000 here or just an arbitrary timeout?

Reply:
It took my machine about 4 seconds to run the command, so I made it 5 just to be safe.

Reply:
hm. that seems like we'll be cutting it pretty close, especially if anyone is using Blitz on a computer with specs slower than yours. Can we make this 10-15s?

Reply:
Sure, 10s seems reasonable.